mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafText.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  LeafText.swift
+//  
+//
+//  Created by Cris Messias on 06/10/23.
+//
+
+import SwiftUI
+
+//MARK: - TitleModifier
+
+public struct TitleModifier: ViewModifier {
+    @Environment(\.leafTheme) private var theme
+    public func body(content: Content) -> some View {
+        content
+            .font(theme.font.titleLarge)
+            .bold()
+            .lineSpacing(0.2)
+            .foregroundStyle(theme.color.content.primary)
+
+    }
+}
+public extension View {
+    func titleModifier() -> some View {
+        self.modifier(TitleModifier())
```

Comment:
We should define a standard regarding the use of `self`. You are using it here but not in the above function when you call `theme`. Usually I prefer not to use self unless it is required by code, I think we keep code more clean that way. What do you think?

Reply:
These are the last two changes that I couldn't fix. I will put a PR to you see the code and help me.